terraform/aws/analytical-platform-management-production/aws-secrets-manager/terraform.tf

```diff
@@ -0,0 +1,53 @@
+terraform {
+  backend "s3" {
+    acl            = "private"
+    bucket         = "global-tf-state-aqsvzyd5u9"
+    encrypt        = true
+    key            = "aws/analytical-platform-management-production/aws-secrets-manager/terraform.tfstate"
+    region         = "eu-west-2"
+    dynamodb_table = "global-tf-state-aqsvzyd5u9-locks"
+
+  }
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "6.8.0"
```

Comment:
Can this be updated please? https://github.com/hashicorp/terraform-provider-aws/releases / https://registry.terraform.io/providers/hashicorp/aws/latest